components/playback/controls/StimulationStudioPlateWell.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <div>
+    <div
+      class="div__simulationstudio-plate-well-location"
+      :style="'top:' + computed_top + 'px;' + 'left:' + computed_left + 'px;'"
+    >
+      <PlateWell
+        :classname="classname"
+        :svg_height="svg_height"
+        :svg_width="svg_width"
+        :circle_x="circle_x"
+        :circle_y="circle_y"
+        :radius="radius"
+        :strk="strk"
+        :plate_fill="protocol_fill"
+        :stroke_wdth="stroke_wdth"
+        :index="index"
+      ></PlateWell>
+    </div>
+    <span
+      class="span__simulationstudio-plate-well-protocol-location"
+      :style="
+        'top:' +
+        computed_protocol_top +
+        'px;' +
+        'left:' +
+        computed_protocol_left +
+        'px;'
+      "
+    >
+      {{ protocol_type }}
+    </span>
+  </div>
+</template>
+<script>
+import PlateWell from "@/components/playback/controls/PlateWell.vue";
+
+export default {
+  name: "StimulationStudioPlateWell",
+  components: {
+    PlateWell,
+  },
+  props: {
+    classname: { type: String, default: "" },
+    svg_height: { type: Number, default: 0 },
+    svg_width: { type: Number, default: 0 },
+    circle_x: { type: Number, default: 0 },
+    circle_y: { type: Number, default: 0 },
+    radius: { type: Number, default: 0 },
+    strk: { type: String, default: "" },
+    protocol_fill: { type: String, default: "" },
+    stroke_wdth: { type: Number, default: 0 },
+    index: { type: Number, default: 0 },
+    protocol_type: { type: String, default: "" },
+  },
+  computed: {
+    /*   0 4  8 12 16 20     In order to speed the rendering its better to pre-compute
```

Comment:
this is 130 lines of code instead of ~3 lines that would do the calculations through equations (similar to `PlateNavigator`).  can you elaborate on the benefit you expect in performance that would justify the large increase in the number of lines of code to read through?

Reply:
Done.